### PR TITLE
W-14139140 added doc about deploy and redeploy flags

### PR DESCRIPTION
**Pull request by AxMuller**
Added Java version, Release channel description to deploy and redeploy command

### CODE REVIEW

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 1]
```diff
@@ -182,12 +182,20 @@ In addition to the default flag `--help`, this command accepts the following fla
 | The CloudHub Gateway version
 | `--gatewayVersion: 9.9.9.9  643404`
 
+| `--javaVersion`
+| Gateway Java version
+| `--javaVersion 17 643404`
+
+| `--releaseChannel`
+| Release channel
+| `--releaseChannel EDGE 643404`
```
Comment (hernanrivero) [inline modules/ROOT/pages/_partials/api-mgr.adoc]: Can you specify for which deployment types is it used?

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 2]
```diff
@@ -492,12 +500,21 @@ Include to redeploy APIs from unclassified environments | `--environmentName mul
 | `--gatewayVersion`
 | CloudHub Gateway version | `--gatewayVersion 9.9.9 643404`
 
+| `--javaVersion`
+| Gateway Java version
+| `--javaVersion 17 643404`
+
+| `--releaseChannel`
+| Release channel
+| `--releaseChannel EDGE 643404`
```
Comment (hernanrivero) [inline modules/ROOT/pages/_partials/api-mgr.adoc]: Same

File: modules/ROOT/pages/_partials/api-mgr.adoc (hunk 1 above)
Comment (AxMuller) [reply modules/ROOT/pages/_partials/api-mgr.adoc]: sure!

**Review by hernanrivero — DISMISSED** [review]

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 3]
```diff
@@ -492,12 +502,23 @@ Include to redeploy APIs from unclassified environments | `--environmentName mul
 | `--gatewayVersion`
 | CloudHub Gateway version | `--gatewayVersion 9.9.9 643404`
 
+| `--javaVersion`
+| Gateway Java version +
+This flag is used by `--target` `RTF`, `CH`, and `CH2`
```
Comment (failup) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
This flag only works if the target flag is set as `RTF`, `CH`, or `CH2`

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 4]
```diff
@@ -492,12 +502,23 @@ Include to redeploy APIs from unclassified environments | `--environmentName mul
...
+| `--javaVersion 17 643404`
+
+| `--releaseChannel`
+| Set the name of the release channel to be used for the selected Mule version. + Supported values are `NONE`, `EDGE`, and `LTS` +
+This flag is used by `--target` `RTF`, `CH`, and `CH2`
```
Comment (failup) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
This flag only works if the target flag is set as `RTF`, `CH`, or `CH2` 
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 5]
```diff
@@ -182,12 +182,22 @@ In addition to the default flag `--help`, this command accepts the following fla
...
+
 | `--overwrite`
-| Update application if it exists +
-| `--overwrite  643404`
+| Update application if exists +
+Include the flag to enable it | `--overwrite`
 
 | `--target`
-| Hybrid or RTF deployment target ID +
+| Hybrid, RTF, CH or CH2 deployment target ID
```
Comment (failup) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Hybrid, RTF, CH, or CH2 deployment target ID
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 6]
```diff
@@ -182,12 +182,22 @@ In addition to the default flag `--help`, this command accepts the following fla
...
+
+| `--releaseChannel`
+| Set the name of the release channel to be used for the selected Mule version. + Supported values are `NONE`, `EDGE`, and `LTS` +
+This flag is used by `--target` `RTF`, `CH`, and `CH2`
+| `--releaseChannel EDGE 643404`
+
 | `--overwrite`
-| Update application if it exists +
-| `--overwrite  643404`
+| Update application if exists +
```
Comment (failup) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Update application if it exists +
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 7]
```diff
@@ -492,12 +502,23 @@ Include to redeploy APIs from unclassified environments | `--environmentName mul
...
+This flag is used by `--target` `RTF`, `CH`, and `CH2`
+| `--releaseChannel EDGE 643404`
+
 | `--overwrite`
 | Update application if exists +
 Include the flag to enable it | `--overwrite`
 
 | `--target`
-| Hybrid or RTF deployment target ID | `--target 1598794 643404`
+| Hybrid, RTF, CH or CH2 deployment target ID
```
Comment (failup) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Hybrid, RTF, CH, or CH2 deployment target ID 
```

**Review by failup — APPROVED** [review]

**Pull request merged**